src/librustdoc/html/static/main.js

```diff
@@ -1490,6 +1490,8 @@ function hideThemeButtonState() {
     searchState.setup();
 }());
 
+let reset_button_timeout;
```

Comment:
I really don't like global variables like this. Instead, please wrap `copy_path`  into an anonymous function like we do in multiple places here and move the `reset_button_timeout` declaration there.

Another thing: even though it's valid to send an undefined variable to `clearTimeout`, please initialize your value to `null` and check it before calling `clearTimeout`.